skill-tree /src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/skilltree
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
+spring.datasource.username=${MYSQL_DB_USERNAME}
+spring.datasource.password=${MYSQL_DB_PASSWORD}
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
in prod, ddl rules must be `none`. 
It comes at a cost though.
Link: https://stackoverflow.com/questions/42135114/how-does-spring-jpa-hibernate-ddl-auto-property-exactly-work-in-spring

Reply:
okay, we will be creating a separate application-prod.properties